examples/trading/agent.py

```diff
@@ -29,10 +61,15 @@
 
 # Prepare a Web3 specific toolkit, relevant for the trading agent
 tools = [
-    market_data_devnet.fetch_mock_market_data,
-    raydium_cpmm.BuyTokenWithSolTool(),
+    coingecko.FetchMarketDataPerCategoriesTool(),
+    coingecko.GetCoinMarketDataTool(),
+    coingecko.GetCoinHistoricalDataTool(),
+    dexscreener.get_token_data,
```

Comment:
a bit weird that just one tool is not a class

Reply:
unified the formats